src/core/dbus-execute.c

```diff
@@ -1792,6 +1841,56 @@ int bus_exec_context_set_transient_property(
         if (streq(name, "LogRateLimitBurst"))
                 return bus_set_transient_unsigned(u, name, &c->log_ratelimit_burst, message, flags, error);
 
+        if (streq(name, "LogFilterPatterns")) {
+                _cleanup_free_ char **allow_list = NULL, **deny_list = NULL;
```

Comment:
Please comment `_cleanup_free_` is the correct cleanup attribute, instead of `_cleanup_strv_free_`.